src/openrct2-ui/windows/EditorInventionsList.cpp

```diff
@@ -909,7 +909,13 @@ static void window_editor_inventions_list_drag_moved(rct_window* w, sint32 x, si
 {
     rct_research_item *researchItem;
 
-    researchItem = get_research_item_at(x, y);
+    // Skip always researched items, so that the dragged item gets placed underneath them
+    do
+    {
+        researchItem = get_research_item_at(x, y);
+        y += LIST_ROW_HEIGHT;
+    } while (researchItem != nullptr && research_item_is_always_researched(researchItem));
```

Comment:
This needs a bit more work. `get_research_item_at` does not return `nullptr` when at the end of the list, but an entry with vlaues like `-1` and `0xFF`.

Reply:
Are you really sure about that, @Broxzier ? `get_research_item_at` takes window coordinates and can return nullptr.

Reply:
Quite sure, yes. It returns `nullptr` when clicked outside of the scroll areas, but when inside the scroll area, it returns the separator.